### DISCUSSION TITLE
Problems with 116 version

### DISCUSSION THREAD

**Discussion by axm1** [ultrafunkamsterdam/undetected-chromedriver#1482] [General]
it happens at today with the latest chrome version

**Comment by m3az88** [comment]
any solution for that?

**Comment by keysenYu** [comment]
+1   help!!!

**Comment by dexedrine-01** [comment]
+1

**Comment by Mtsumi** [comment]
Thank God for this discussion. I was wondering what just happened? Any solution yet?

**Comment by mshot** [comment]
u can try 
https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1475#issuecomment-1680925178
or try fix with something like this PR
https://github.com/ultrafunkamsterdam/undetected-chromedriver/pull/1478

**Comment by axm1** [comment]
from seleniumbase import Driver
browser = Driver(uc=True)

this library is not undetected_chromdriver, but it works for me

**Comment by mk-045** [comment]
help

**Comment by User2033** [comment]
+1

selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:63568
from session not created: This version of ChromeDriver only supports Chrome version 114
Current browser version is 116.0.5845.97

**Comment by brunoruas2** [comment]
This issue has many ways to overcome this problem

https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1491

I just tried the first one and worked!

**Comment by zakcali** [comment]
now 117 is a problem:
from session not created: This version of ChromeDriver only supports Chrome version 117
Current browser version is 116.0.5845.188

**Comment by zakcali** [comment]
I've just upgraded my chrome version to 117 and problem solved
